polar-core/src/vm.rs

```diff
@@ -1485,7 +1550,7 @@ impl PolarVirtualMachine {
                     term,
                     format!(
                         "{} isn't something that is true or false so can't be a condition",
-                        term.value().to_polar()
+                        term.value()
```

Comment:
This is probably what caused the test suite regression noted elsewhere. Do we need the `.value()` call for some reason I'm not seeing?

```suggestion
                        term
```